### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by jklenzing**
Initial version.  Code directly imported from pysat v2.0.0 (dev).

### CODE REVIEW

**Comment by jklenzing** [comment]
This one is ready to go too.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019, RS
+# Full license can be found in License.md
+# -----------------------------------------------------------------------------
+
+import os
+from codecs import open
```
Comment (aburrell) [inline setup.py]: Recommend not renaming existing function, `open`.

Comment (jklenzing) [reply setup.py]: Code updated.

**Comment by jklenzing** [comment]
Updated codecs usage, ready for review.

**Pull request merged**